Azure/sonic-buildimage
# [DellEMC]: Start with the port index 0 instead of 1 in all DellEMC platform HWSKU's.

**- What I did**
Changed the port index in port_config.ini to maintain uniformity.
If this is not done,we faced the following issue in few platforms.

SFP index 32 out of range (0-31) 

Traceback (most recent call last): 

File "/usr/bin/xcvrd", line 1080, in 
 main() 
File "/usr/bin/xcvrd", line 1077, in main 
   xcvrd.run() 
File "/usr/bin/xcvrd", line 1041, in run 
    self.init() 
 File "/usr/bin/xcvrd", line 1025, in init 
    post_port_sfp_dom_info_to_db(is_warm_start, self.stop_event) 
 File "/usr/bin/xcvrd", line 388, in post_port_sfp_dom_info_to_db 
    post_port_sfp_info_to_db(logical_port_name, int_tbl, transceiver_dict, stop_event) 
 File "/usr/bin/xcvrd", line 224, in post_port_sfp_info_to_db 
    if not _wrapper_get_presence(physical_port): 
 File "/usr/bin/xcvrd", line 123, in _wrapper_get_presence 
    return platform_chassis.get_sfp(physical_port).get_presence() 
AttributeError: 'NoneType' object has no attribute 'get_presence' 

**- How I did it**
get_sfp() is common for all platforms. 
In 1.0,get_presence()is handled in sfputil.py which is platform specific file and port indexing is required. 
Change port_config.ini file.
**- How to verify it**
Generate config_db.json using sonic-cfggen with the new HWSKU and reload the config.
Check whether all the CLI's are working properly.
**- Description for the changelog**
Change the indices alone.
